<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caracteristiques</class>
 <widget class="QDialog" name="Caracteristiques">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tout">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="inventaire">
      <attribute name="title">
       <string>Inventaire</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="lay_touteslesvus">
         <item>
          <layout class="QHBoxLayout" name="lay_vuobjet">
           <item>
            <widget class="QFrame" name="objets_gauche">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="lay_vu">
             <item>
              <widget class="QLabel" name="vuperso">
               <property name="text">
                <string>Vue du perso</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="lay_argpod">
               <item>
                <widget class="QLabel" name="argent">
                 <property name="text">
                  <string>argent</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QProgressBar" name="pod">
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="objets_droite">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="lay_nivetnom">
           <item>
            <widget class="QLabel" name="nomobjet">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="nivobjet">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="lay_obj">
           <item>
            <layout class="QVBoxLayout" name="lay_podimage">
             <item>
              <widget class="QLabel" name="podobjet">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="imageobjet">
               <property name="text">
                <string>Séléctionnez un objet.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTabWidget" name="tab_effetcondition">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="attaque">
              <attribute name="title">
               <string>Attaque</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_3"/>
             </widget>
             <widget class="QWidget" name="effet">
              <attribute name="title">
               <string>Effet</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QTextEdit" name="descr_effet"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="conditions">
              <attribute name="title">
               <string>Conditions</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QTextEdit" name="descr_condition"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="description">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="lay_inventaire"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="caracteristiques">
      <attribute name="title">
       <string>Caractéristiques</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="lay_nom_lvl_classe">
         <item>
          <widget class="QLabel" name="nom">
           <property name="font">
            <font>
             <family>Droid Sans Arabic</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Nom</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="classe_lvl">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Classe et lvl</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelXp">
           <property name="text">
            <string>xp :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="barre_xp">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="text_pointscara">
         <property name="text">
          <string>Nombre de points caractéristiques :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tab_cara">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <row>
          <property name="text">
           <string>Force</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Points fight</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Vie</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Base</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Bonus</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="1">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="1">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="2">
          <property name="flags">
           <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="competences">
      <attribute name="title">
       <string>Compétences</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTabWidget" name="tab_metiers">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetSort">
      <attribute name="title">
       <string>Sorts</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
